pom.xml

```diff
@@ -81,12 +81,13 @@
         <hibernate-search.version>7.2.3.Final</hibernate-search.version>
 
         <!-- Make sure to check compatibility between these 2 gRPC components before upgrade -->
-        <grpc.version>1.69.1</grpc.version> <!-- when updating, verify if com.google.auth should not be updated too -->
+        <grpc.version>1.69.1</grpc.version> <!-- when updating, verify if com.google.auth and perfmark.version should not be updated too -->
```

Comment:
Ideally, could you add a link to https://central.sonatype.com/ on the artifact where we can check the versions? Sometimes it's quite hard to figure out when you have to navigate through a couple of parent POMs.